### ISSUE TITLE
handle missing imports

### ISSUE THREAD

**Issue by Leryan** [SublimeLinter/SublimeLinter-govet#10]
This simple code doesn't trigger anything on the linter:

```go
package main

func main() {
	fmt.Println("truc")
}
```

This is likely related to `govet`:

```
$ govet -all -v .
vet: main.go:4:2: undeclared name: fmt
Checking file main.go
```

In fact, any `undeclared name`, like unknown variables (typos for example) should trigger the linter maybe?

**Comment by kaste** [comment]
Hi @Leryan 

Currently we actually just run `go tool vet <file.go>`. Maybe that's stupid/outdated, but we don't have a lot of go people in the SublimeLinter org. So we really appreciate community input here.

**Comment by Leryan** [comment]
I'll check what is available to do on that point asap, thanks for the infos

**Comment by chilic** [comment]
PR https://github.com/SublimeLinter/SublimeLinter-govet/pull/11 should solve this issue.
